Missing libgl in link flags

There is a missing link flag in the Makefile that result in the following error:
```make
/usr/bin/ld: /usr/lib/gcc/x86_64-linux-gnu/4.9/../../../../lib/libglfw.so: undefined reference to symbol 'glGetString'
```
Adding the appropriate -lGL in LDFLAGS makes it work:
```make
LDFLAGS= -Wl,--no-as-needed -lglfw -lGL -lGLEW -lIL -lSOIL
```

--- a/Makefile
+++ b/Makefile
@@ -1,14 +1,14 @@
 CXX=clang++
 CXXFLAGS= -O3 -g -ggdb -Wall -Wextra -pedantic -Wno-local-type-template-args -Wno-unused-parameter -Wno-logical-op-parentheses -std=c++11
-LDFLAGS= -lglfw -lGLEW -lGL -lIL -lSOIL
+LDFLAGS= -lglfw -lGL -lGLEW -lIL -lSOIL
 
 CXX = g++
 CXXFLAGS=-Wall -Wextra -Wno-unused-parameter -O0 -g -ggdb -p -pg -std=c++11 -pedantic -Wstrict-aliasing -DDEBUG
-LDFLAGS= -Wl,--no-as-needed -lglfw -lGLEW -lIL -lSOIL
+LDFLAGS= -Wl,--no-as-needed -lglfw -lGL -lGLEW -lIL -lSOIL
 
 CXX = g++
 CXXFLAGS=-Wall -Wextra -Wno-unused-parameter -O3 -std=c++11 -pedantic -Wstrict-aliasing
-LDFLAGS= -Wl,--no-as-needed -lglfw -lGLEW -lIL -lSOIL
+LDFLAGS= -Wl,--no-as-needed -lglfw -lGL -lGLEW -lIL -lSOIL
 
 all: demo
 

